### PR TITLE
#197 Updated the Home view to look like the vue version.

### PR DESCRIPTION
**Pull request by ed-snodgrass**
Signed-off-by: Ed Snodgrass <snodgrass.ed@gmail.com>

Contributes to # (add issue number here. i.e. `#302`)

## What did you do?
Added a home view with a hero image with text overlayed.

## Why did you do it?
Last piece of React Admin.

## How have you tested it?
Loaded the page

## Were docs updated if needed?

- [ ] No
- [ ] Yes
- [x] N/A

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new console logs
- [x] Fixes entire issue
- [ ] Partial fix for issue

### CODE REVIEW

**Comment by markstur** [comment]
Hi @ed-snodgrass,  You probably noticed already.  But FYI... failed tests.

**Comment by ed-snodgrass** [comment]
> Hi @ed-snodgrass, You probably noticed already. But FYI... failed tests.

I had not noticed. Thank you @markstur and I apologize for that. I've re-added the testid and we should be good to go.

File: server/react-admin/src/sidemenu/SideNav.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ const SideNav = ({ open, handleClose, handleOpen }) => {
     { label: 'Officials', link: '/officials' },
     { label: 'Publications', link: '/publications' },
     { label: 'Video Testimonials', link: '/videoTestimonials' },
-    { label: 'REST API Docs', link: '/api-docs' },
+    { label: 'REST API Docs', link: 'http://localhost:5000/api-docs' },
```
Comment (markstur) [inline server/react-admin/src/sidemenu/SideNav.js:52]: ideally the PORT would never be hard-coded, but no harm for now

**Review by markstur — APPROVED** [review]
I was amazed at how it looked exactly like the old version and then I remembered the new code lives under react-admin (not admin).

It looks good.  One comment on the hard-coded port 5000 for the swagger.  Can deal with that later (configurable, proxy, or just remove it from the UI).

Great to have this done.  Thanks, Ed!

**Pull request merged**